server.py

```diff
@@ -0,0 +1,106 @@
+import argparse
+import asyncio
+import configparser
+import grp
+import json
+import mimetypes
+import multiprocessing
+import os
+import pwd
+import sys
+import time
+import uuid
+from concurrent.futures import ProcessPoolExecutor
+from urllib.parse import urlparse, unquote, parse_qsl
+from versions_manager import VersionManager
+import aiohttp
+import git
+import pip
+from aiohttp import web
+import re
+import logging
+import logger
+import multidict
+import aiohttp_jinja2
+import jinja2
+
+try:
+    from aiohttp.web import StaticResource as StaticRoute
+except ImportError:
+    from aiohttp.web import StaticResource
+
+from bs4 import BeautifulSoup
+import cssutils
+import netifaces as ni
+from converter import Converter
+from middlewares import SnareMiddleware
+from tanner_handler import TannerHandler
+
+class HttpRequestHandler():
+    def __init__(self, meta, run_args, snare_uuid, debug=False, keep_alive=75, **kwargs):
+        self.run_args = run_args
+        self.dir = '/opt/snare/pages/{}'.format(run_args.page_dir)
+        self.meta = meta
+        self.snare_uuid = snare_uuid        
+        self.logger = logging.getLogger(__name__)
+        self.sroute = StaticRoute(
+            name=None, prefix='/',
+            directory=self.dir
+        )
+        self.tanner_handler = TannerHandler(run_args, meta, snare_uuid)
+
+    async def handle_request(self, request):
+        self.logger.info('Request path: {0}'.format(request.path))
+        data = self.tanner_handler.create_data(request, 200)
+        if request.method == 'POST':
```

Comment:
maybe we can split post and get handlers?

Reply:
what about PUT and other requests? If we split both handlers, we need to define handlers for others HTTP request as well. What do you think about it?